build/configs/common_download.sh

```diff
@@ -433,11 +433,18 @@ IFS=',' read -ra sizes <<< "$sizes"
 #Calculate Flash Offset
 num=${#sizes[@]}
 offsets[0]=`printf "0x%X" ${CONFIG_FLASH_START_ADDR}`
-
+flag=0
```

Comment:
What's the meaning of `0` and `1`? Let's use some readable word instead of 0 and 1.
or let's use other name instead of flag .

Reply:
This change is not needed now.